tools/packaging/packaging.psm1

```diff
@@ -22,11 +22,6 @@ function Start-PSPackage {
         [ValidateSet("deb", "osxpkg", "rpm", "msi", "zip", "AppImage", "nupkg")]
         [string[]]$Type,
 
-        # Generate windows downlevel package
```

Comment:
My bad, I forgot that we are doing cross compilation so the actual RID will still be `*-x64`.
@adityapatwardhan and I just talked and agreed that we should bring back the `WindowsRutnime` parameter here. And that parameter probably should be made mandatory.